packages/react-microfrontend/src/fetch-retry.js

```diff
@@ -0,0 +1,26 @@
+const LIMIT = 1;
+
+const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms));
+const fetchRetry = async (url, { limit = LIMIT, delay, ...opts } = {}) => {
+  try {
+    const response = await fetch(url, opts);
+    const { status, statusText } = response;
+
+    if (status >= 200 && status < 300) {
+      return response.json();
+    }
+
+    throw new Error(statusText)
```

Comment:
I think errors should be used to errors only and not to create changes on behaviour. Maybe in this line I would remove and move most of logic in catch statement to outside.